src/frontends/common/include/openvino/frontend/frontend.hpp

```diff
@@ -143,6 +143,9 @@ class FRONTEND_API FrontEnd {
         add_extension(args...);
     }
 
+    /// \brief TODO: UPDATE
+    virtual void configure(const ov::AnyMap& config);
```

Comment:
I would like to have aligned API with plugins (`set_property()`).

Reply:
Moved it to https://github.com/openvinotoolkit/openvino/pull/16335